packages/mobile/ios/celo/AppDelegate.m

```diff
@@ -84,6 +88,20 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
   self.window.rootViewController = rootViewController;
   [self.window makeKeyAndVisible];
 
+  // register for push notifications
+  UNUserNotificationCenter* center = [UNUserNotificationCenter currentNotificationCenter];
+  center.delegate = self;
+  [center requestAuthorizationWithOptions:(UNAuthorizationOptionAlert | UNAuthorizationOptionSound | UNAuthorizationOptionBadge)
+  completionHandler:^(BOOL granted, NSError * _Nullable error) {
+    if (granted) {
+      dispatch_async(dispatch_get_main_queue(), ^(void) {
+        [[UIApplication sharedApplication] registerForRemoteNotifications];
+      });
+    }
+  }];
```

Comment:
This is gonna trigger the push notification permission popup directly on the first app launch.
I don't think we want this. Right now it happens when creating/restoring an account.

Can we remove this?

Reply:
if it's happening later, yes it can be removed :)